reactDashboard/src/components/DataCharts/DataChartsUtils.js

```diff
@@ -111,6 +111,7 @@ export function commonChartConfiguration(datasets) {
       },
       legend: {
         position: "bottom",
+        onClick: function (event, legendItem) {},
```

Comment:
Yup. This page https://stackoverflow.com/questions/38583894/how-to-disable-chartjs-legendclick suggests a modification:
`legend: { onClick: (e) => e.stopPropagation() }`

Reply:
Modified that line